petastorm/tests/test_common.py

```diff
@@ -159,7 +159,8 @@ def _partition_key_to_str(row):
     return dataset_dicts
 
 
-def create_test_scalar_dataset(output_url, num_rows, num_files=4, spark=None, partition_by=None):
+def create_test_scalar_dataset(output_url, num_rows, num_files=4, spark=None,
+                               partition_by=None, two_datetime_vals=False):
```

Comment:
Can you please clarify why did you choose to use alternating datatime values instead of a condition on an existing field? I think our test fixture data is not very well structured and hard to use (different fields are set using not clear nor intuitive logic).Can your test rely on one of the existing fields instead of adding more complexity to the data (unless you think your solution simplifies the testing code by alot)?

Reply:
modified to use an existing field.